custom/icds_reports/tests/agg_tests/reports/test_awc_reports.py

```diff
@@ -2268,15 +2268,15 @@ def test_awc_report_lactating_first_record(self):
                 data['data'][0],
                 {
                     'num_rations_distributed': 0,
-                    'institutional_delivery': 'N',
+                    'institutional_delivery': 'No',
```

Comment:
probably not worth it now, but if these had been extracted to a constant then you'd only have to have edited 3 letters :upside_down_face: